activerecord/lib/active_record/serialization.rb

```diff
@@ -10,8 +10,8 @@ module Serialization
       self.include_root_in_json = false
     end
 
-    def serializable_hash(options = nil)
-      options = options ? options.dup : {}
+    def serializable_hash(options = {})
+      options = options.dup
```

Comment:
There's now a second allocation here when `as_json` is called with no arguments.

Reply:
👍 Corrected.